README.md

```diff
@@ -14,13 +14,13 @@ Lxc is similar to docker but aims more to be an OS container instead of just app
 To use it, install lxd and initialize it using `lxd init`. When prompted, answer the following questions:
 
   - Would you like to use LXD clustering? (yes/no) [default=no]:        
-  - Do you want to configure a new storage pool? (yes/no) [default=yes]: yes -> default location
+  - Do you want to configure a new storage pool? (yes/no) [default=yes]: yes -> **dir** 
```

Comment:
I think what you mean is that one should use some directory based storage provider, right? Would you mind adding a few more words about this?